GQM: Adoption trend and related metric (#78)
InnerSourceCommons/managing-innersource-projects: measuring/questions/adoption-trend.md

[⬑ back to the overall graph](../use_gqm.md)

# **Question:** What is the InnerSource adoption trend?

> This question is something an InnerSource program asks while fostering the adoption of InnerSource.

Reviewer: ```suggestion
```
Is this redundant? 

As ISPOs, do we assume we always foster adoption of InnerSource?

Author: Hi @jeffabailey , the idea is that ISPOs may focus on different types of tasks overtime, e.g., fostering adoption vs raising the maturity of existing adopters. The trend of adoption is one information that can help the teams prioritize one over the other.

Lack of adoption, or stagnant adoption may reveal that it is necessary to rethink the program strategy or investigate impediments.

## Related Metrics

| **Metric** | **How it answers the question** | **Gotchas** |
| --- | --- | --- |

> | [Number of InnerSource repositories](../metrics/number-of-innersource-repositories.md) | The number of (source code) repositories that are consider InnerSource are directly related to the adoption of the practice. | Depending on how you define _InnerSource repositories_, it can be prone to false positives and/or negatives |

Reviewer: ```suggestion
| [Number of InnerSource repositories](../metrics/number-of-innersource-repositories.md) | The number of (source code) repositories that are directly related to the adoption of the practice. | Depending on how you define _InnerSource repositories_, it can report incorrect results |
```

Here's a little more concise version.

I detected a small grammar issue.

> | [Number of InnerSource repositories](../metrics/number-of-innersource-repositories.md) | The number of (source code) repositories that are consider InnerSource are directly related to the adoption of the practice. | Depending on how you define _InnerSource repositories_, it can be pronte to false positives and/or negatives |

Reviewer 2: did you mean "prone"?
